lutris/runners/winesteam.py

```diff
@@ -156,7 +156,8 @@ def launch_args(self):
         args = [self.get_executable(), self.get_steam_path()]
 
         # Fix invisible text in Steam
-        args.append('-no-dwrite')
+        # Since Wine 1.7.50, DirectWrite has been fixed
+        # args.append('-no-dwrite')
```

Comment:
If so, just remove the code entirely. (This should be separate commit though, so you can explain the reasoning properly.) The code will still be in the git repository, so there's no need to leave commented out stale code in the codebase.